plugins/modules/remote_management/manageiq/manageiq_tags.py

```diff
@@ -60,6 +64,21 @@
       username: 'admin'
       password: 'smartvm'
       validate_certs: False
+      
+- name: Create new tags for a provider in ManageIQ
+  community.general.manageiq_tags:
+    resource_id: 23000000790497
```

Comment:
Just a reminder: due to regions, manageiq object ids may be so large that when represented as JSON number they might be parsed wrong due to floating-point rounding.
This affects JS which has no bigints, only floating-point:
```javascript
> JSON.parse('{"id": 9923000000790497}')
{ id: 9923000000790496 }
```
And the solution adopted was to represent ids as strings rather than numbers in JSON.
But in these modules we're calling Ruby from Python.  Both have bigints:
```ruby
irb(main):004:0> JSON.load('{"id": 999999999923000000790497}')
=> {"id"=>999999999923000000790497}
```
And in these 2 modules, the id is only going into the URL, not JSON, so it's a string anyway.
So all is good :heavy_check_mark:

Reply:
Thanks for the insight.